style: apply /style-guide pass to weave (#2739)
wandb/docs: weave/guides/integrations/agno.mdx

)
```

> The reasoning steps are visible in the trace, which shows how the agent breaks down complex problems and produces decisions.

Reviewer: ```suggestion
The trace displays the reasoning steps that show how the agent breaks down complex problems and makes decisions.
```

Author: Applied verbatim in eaf82665d.
